certification/pyxis/submit.go

```diff
@@ -23,17 +25,23 @@ func (p *pyxisEngine) SubmitResults(certProject *CertProject, certImage *CertIma
 		}
 	}
 
+	dockerImageDigest := certImage.DockerImageDigest
+
 	certImage, err = p.createImage(ctx, certImage)
-	if err != nil {
+	if err != nil && err != errors.Err409StatusCode {
 		log.Error(err, "could not create image")
 		return nil, nil, nil, err
+	} else {
```

Comment:
There is no need for the else is there?

Reply:
Or rather... Perhaps you should check if the error IS a 409, and do the thing, otherwise, return.

Reply:
went with two `if's` approach, lmk what you think.